scripts/pre_commit.R

```diff
@@ -11,3 +11,5 @@ do.call(styler::style_dir, c(list(path = here::here("tests")), styler_confs))
 source(here::here("reference_data", "build_package_data.R"))
 devtools::document()
 devtools::test()
+devtools::uninstall()
+devtools::install()
```

Comment:
Good point. I had a very hard time with the install command so maybe I am doing something wrong. I can successfully install the package exactly once at the beginning of a new session, but whenever I change something in one of the function and run it again, even with uninstall before it, I get a strange database error that some state or so is not consistent. So I think I added it here but I don't exactly remember why because yeah it is not needed for styler and also not for the tests or documentation....

Reply:
Ah I remember I wanted to make sure I have always the latest correct package installed. I first had this in the Rprofile file, but the start of Rstudio just takes too long with this so I switched back to only call devtools::load_all() at the start of a new session